src/App.css

```diff
@@ -522,11 +523,23 @@ a.text-link {
   transition: transform 0 ease-out, margin 0 ease-out; /* add transition time after initial load and scroll into place */
   overflow: hidden; /* prevent expanded click areas from adding scroll-x */
   box-sizing: border-box; /* make height include padding so that the min-height can be exctly 100vh */
-  max-width: 50em;
+  max-width: 66%;
   margin: 0 auto;
   min-height: 100vh; /* forces footer to bottom when there is little content */
 }
 
+@media (max-width: 960px) {
```

Comment:
Hi @raineorshine ,
I think it would be good if you defined global mixins for breakpoints such that when someone wants to design for mobile or tablet there is a predefined breakpoint such that they would just use the `@include respond-to(the-device)`
What do you think?

Reply:
Hey! Thanks for the suggestion. I am not familiar with this approach, but I am open to it. I actually think we're going to be transitioning away from the CSS file to use inline styles which will be necessary to convert to React Native.